gateway/src/node/client_handling/active_clients.rs

```diff
@@ -126,6 +130,9 @@ impl ActiveClientsStore {
     /// * `client`: address of the client for which to remove the handle.
     pub(crate) fn disconnect(&self, client: DestinationAddressBytes) {
         self.inner.remove(&client);
+        let _ = self
+            .stats_event_sender
+            .unbounded_send(events::new_session_stop_event(client));
```

Comment:
I know this was likely copying the pattern from surrounding code, but it was helpfully pointed out to me during another review that we should log errors for stuff like this, and thinking about it I kinda agree.
Imho let's log an error as a defensive action

Reply:
Fixed in adf198f6b8696b963643253dd0ee3aa6e745f13c